docs/excel/excel-add-ins-pivottables.md

```diff
@@ -221,7 +221,128 @@ Excel.run(function (context) {
 });
 ```
 
-## Slicers
+## Filter a PivotTable
+
+The Excel JavaScript API offers two methods for filtering PivotTable data: PivotFilters and slicers. [PivotFilters](/javascript/api/excel/excel.pivotfilters) allow you to filter data based on a PivotTable's four [hierarchy categories](#hierarchies) (filters, columns, rows, and values). [Slicers](/javascript/api/excel/excel.slicer) can be applied to both PivotTables and regular Excel tables. The `Slicer` class, like [its Excel UI counterpart](https://support.office.com/article/Use-slicers-to-filter-data-249f966b-a9d5-4b0f-b31a-12651785d29d), allows for style and formatting customization. `PivotFilters` do not offer style or formatting flexibility. 
```

Comment:
Somewhere, this article needs to address the relationship between PivotFilters and a PivotHierarchy. That's hard to determine just through the ref docs and a great candidate for a conceptual article topic.